rpc-server/src/errors.rs

```diff
@@ -92,3 +92,56 @@ where
         }
     }
 }
+
+#[derive(thiserror::Error, Debug, Clone)]
+pub enum CallFunctionError {
```

Comment:
It'd be better to stay consistent and name it `FunctionCallError` (word ordering)